### PR TITLE
[Model]Remove Dropout Layers

### PR DESCRIPTION
**Pull request by alex-jw-brooks**
Cleanup to remove some dropout layers / references, which shouldn't be needed in vLLM (still need to remove from phi4mm / minicpm o)

### CODE REVIEW

**Review by jeejeelee — APPROVED** [review]
LGTM

File: vllm/model_executor/models/ultravox.py [hunk 1]
```diff
@@ -366,9 +366,6 @@ def forward(
         embed_pos = self.embed_positions.weight[:inputs_embeds.size(-2)]
 
         hidden_states = inputs_embeds + embed_pos
-        hidden_states = nn.functional.dropout(hidden_states,
-                                              p=self.dropout,
-                                              training=self.training)
```
Comment (DarkLight1337) [inline vllm/model_executor/models/ultravox.py:371]: Also remove training flag here?

Comment (alex-jw-brooks) [reply vllm/model_executor/models/ultravox.py:371]: Hey @DarkLight1337! I think I may misunderstand - `self.training` should always be `False` in vLLM, right?

Comment (DarkLight1337) [reply vllm/model_executor/models/ultravox.py:371]: Yes that is correct
